scitran/data/medimg/dcm/dcm.py

```diff
@@ -235,20 +235,26 @@ def __init__(self, path, load_data=False, timezone=None):
 
         """
         super(Dicom, self).__init__(path, load_data, timezone)
-        with tarfile.open(path) as archive:
-            for ti in archive:
+        zip_dicom = zipfile.ZipFile(path)
```

Comment:
Use a context manager so that the zip file gets closed automatically. Similar to the previous tarfile code.